### ISSUE TITLE
ImportError: cannot import name 'outcome_constraint_from_str' from 'ax.service.utils.instantiation' 

### ISSUE THREAD

**Issue by fangrouPER** [facebookresearch/Kats#216]
I got this error since 2022-03-10:
`ImportError: cannot import name 'outcome_constraint_from_str' from 'ax.service.utils.instantiation' (/databricks/python/lib/python3.7/site-packages/ax/service/utils/instantiation.py)`
Is that related to recent changes ?
https://github.com/facebookresearch/Kats/commit/ef75a9b24e962069e2e4d1ecbe1557f5061b2cbd#diff-78ab96e7252fcd290c23f37663edc2b462c84c09d103017ec2a410685cb25cceL40

Thanks for your reply

```
      5 import pandas as pd
      6 import numpy as np
----> 7 from kats.detectors.bocpd import BOCPDetector, BOCPDModelType
      8 from kats.detectors.bocpd import NormalKnownParameters
      9 from kats.consts import TimeSeriesData

/databricks/python/lib/python3.7/site-packages/kats/__init__.py in <module>
      1 from . import consts   # noqa
----> 2 from . import utils  # noqa
      3 from . import detectors  # noqa
      4 from . import models  # noqa
      5 from . import tsfeatures   # noqa

/databricks/python/lib/python3.7/site-packages/kats/utils/__init__.py in <module>
      5 from . import parameter_tuning_utils   # noqa
      6 from . import simulator   # noqa
----> 7 from . import time_series_parameter_tuning   # noqa

/databricks/python/lib/python3.7/site-packages/kats/utils/time_series_parameter_tuning.py in <module>
     36 from ax.modelbridge.registry import Models
     37 from ax.runners.synthetic import SyntheticRunner
---> 38 from ax.service.utils.instantiation import (
     39     outcome_constraint_from_str,
     40     parameter_from_json,

ImportError: cannot import name 'outcome_constraint_from_str' from 'ax.service.utils.instantiation' (/databricks/python/lib/python3.7/site-packages/ax/service/utils/instantiation.py)
```

**Comment by fangrouPER** [comment]
seems like other people also got similar error :
https://github.com/facebookresearch/Kats/issues/215
https://github.com/facebookresearch/Kats/issues/214
https://github.com/facebookresearch/Kats/issues/213

**Comment by kylemcq13** [comment]
came across this same error this morning - I downgraded ax_platform to 0.2.3 which seems to work

**Comment by iamxiaodong** [comment]
Hello @fangrouPER and @kylemcq13: this is due to a recent change of `ax-platform` library that moved some APIs to a new module, the easiest fix is to install their newest version `v0.2.4`, see the Github page here: https://github.com/facebook/Ax

We're working on releasing a new Kats version which will restrict the `ax` version to be `0.2.4` or higher for the future installation. At this moment, please upgrade your `ax-platform` version and install Kats from source if you prefer. Thanks.

**Comment by fangrouPER** [comment]
> Hello @fangrouPER and @kylemcq13: this is due to a recent change of `ax-platform` library that moved some APIs to a new module, the easiest fix is to install their newest version `v0.2.4`, see the Github page here: https://github.com/facebook/Ax
> 
> We're working on releasing a new Kats version which will restrict the `ax` version to be `0.2.4` or higher for the future installation. At this moment, please upgrade your `ax-platform` version and install Kats from source if you prefer. Thanks.

Thanks @iamxiaodong for informations, the solution proposed by @kylemcq13 works for me: set **ax-platform==0.2.3** in requirements.
